app/services/metrics/defect_escape_rate/per_project.rb

```diff
@@ -27,9 +27,14 @@ def query(interval)
         ]]
       end
 
+      # :reek:DuplicateMethodCall
```

Comment:
I'd prefer to have these comments in `.reek.yml` rather than polluting the actual code with them